### PR TITLE
feat: enums checks for file-level comparator

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: test/testdata/protos/example/enum_v1.proto
Comment (chrisdunelm) [inline test/testdata/protos/example/enum_v1.proto:5]: nit: Don't feel you need to change this, but I think I'd rather there are no meaningful names in these tests. Just call them things like `Enum1` and `Enum2`. Using meaningful names, to my mind, adds a small cognitive overhead!

**Review by chrisdunelm — COMMENTED** [review]
This looks fine to me, although I see it's still in draft so I haven't LGTMed yet.

File: test/testdata/protos/example/enum_v1.proto
Comment (xiaozhenliu-gg5) [reply test/testdata/protos/example/enum_v1.proto:5]: This PR has some duplicate code from #16 so hold this until that one is merged. :)

And sounds good, was also hesitating about the names like fooBar, but I agree I can do a whole unit test name refactor after. Thank you!

Comment (software-dov) [reply test/testdata/protos/example/enum_v1.proto:5]: I like to use Squid and Clam and other molluscs as my test names. I know what kind of attributes a squid has and how it relates to other molluscs, and that helps ground the mental model as reflected by the constructs of the test.

**Review by software-dov — APPROVED** [review]

File: test/testdata/protos/example/enum_v1.proto
Comment (xiaozhenliu-gg5) [reply test/testdata/protos/example/enum_v1.proto:5]: 😄 That's a great advice, when I tried to use `foobar` I'm not sure what to use next.:)
I don't know much about molluscs, but maybe I can pick up some other categories that I'm familiar with.

**Pull request merged**
